heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -189,7 +189,7 @@ private static Options constructOptions() {
         .build();
 
     Option topologyJar = Option.builder("j")
-        .desc("user heron topology jar/pex file path")
+        .desc("user heron topology jar/tar/pex file")
         .longOpt("topology_bin")
         .hasArgs()
         .argName("topology binary file")
```

Comment:
.desc("The remote filename of the heron topology jar/tar/pex file to be run by the executor.")
.argName("topology binary filename on the cluster")